specs/deneb/polynomial-commitments.md

```diff
@@ -336,7 +336,7 @@ def evaluate_polynomial_in_evaluation_form(polynomial: Polynomial,
 
 ### KZG
 
-KZG core functions. These are also defined in Deneb execution specs.
+KZG core functions. These are also defined in Deneb execution specs for EIP-4844.
```

Comment:
The top of this doc mentions "required for Deneb specification". I might be explicit up there that it is for 4844 (within Deneb)